data/human/hails.txt

```diff
@@ -2391,7 +2393,7 @@ phrase "friendly civilian"
 	word
 		"I'm trying to improve my piloting skills without risking my life too much, so I've been mining asteroids."
 		"Mining can be a good way to improve your piloting skills."
-		"Mining is a safe career option for new Sparrow pilots."
+		"Mining is the only safe career option for new Sparrow pilots."
```

Comment:
~~Technically, you can still make a profit by hauling cargo and passengers with a Sparrow, it's just painfully slow.~~